common/persistence/sql/storage/sqldb/interfaces.go

```diff
@@ -463,6 +463,12 @@ type (
 		MessagePayload []byte
 	}
 
+	// QueueMetadataRow represents a row in queue_metadata table
+	QueueMetadataRow struct {
+		QueueType int
```

Comment:
can you plz make the `QueueType` a dedicated type?

Reply:
done